src/Validator/Constraints/RelayUniqueValidator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Validator\Constraints;
+
+use App\Entity\MembreCentre;
+use Doctrine\Common\Collections\ReadableCollection;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class RelayUniqueValidator extends ConstraintValidator
+{
+    /**
+     * @param ReadableCollection<MembreCentre> $value
+     */
+    public function validate(mixed $value, Constraint $constraint)
+    {
+        $duplicates = $this->getDuplicates($value);
+        foreach ($duplicates as $duplicate) {
+            $this->context->buildViolation($constraint->message)
+                ->setParameter('{{ relay }}', $duplicate)
+                ->addViolation();
+        }
```

Comment:
array_map(
            fn (Centre $duplicate) => $this->context->buildViolation($constraint->message)->setParameter('{{ relay }}', $duplicate)->addViolation(),
            $duplicates
        );